src/Servers/IIS/build/testsite.props

```diff
@@ -74,6 +74,9 @@
           Condition=" '$(InProcessTestSite)' == 'true' AND '$(UseIisNativeAssets)' == 'true' AND '$(TargetFrameworkIdentifier)' != '.NETFramework'"
           BeforeTargets="Publish"
           DependsOnTargets="GeneratePublishDependencyFile;CopyFilesToPublishDirectory;PrepareInjectionApp">
-    <Exec Command="$(InjectDepsApp) &quot;$(PublishDepsFilePath)&quot; $(RuntimeIdentifier) " />
+    <!-- Use $(PublishDir)$(ProjectDepsFileName) instead of $(PublishDepsFilePath) here until
+         https://github.com/dotnet/arcade/issues/13911 is fixed.
+    -->
+    <Exec Command="$(InjectDepsApp) &quot;$(PublishDir)$(ProjectDepsFileName)&quot; $(RuntimeIdentifier) " />
```

Comment:
Potentially link https://github.com/dotnet/sdk/blob/b736f2e955a9167eda86cae79208a89a9e42bfe0/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets#L1059 for reference as to where this value comes from

Reply:
yup, will do.